### ISSUE TITLE
Docker Setup is a disaster

### ISSUE THREAD

**Issue by wuast94** [bitwarden/server#3062]
### Steps To Reproduce

Sry for abusing this templated but a forum isnt the right place for this.

The actual docker setup is hilarious. I have to google arround to find an old closed issue where anywhere in the middle is a comment that there is a .zip file with an offline like installer, the link to the docs there are linking to the online installer. using an own reverse proxy seems to be a hassle on its own and so on.

### Expected Result

Have a docker-compose.yml in the repo root where all the services needed are configured.

all configs must be configured with variables in the first place, if i have to edit config files afterwards is a normal thing.

the "one container" for all is not a solution, seeing many requst this, but that nots how containers supposed to be used.

in short: just have an easy streamlined setup with compose like almost any other project out there. would making helping devolopment easier too.

### Actual Result

need google to find all the different informations and a maximal overcomplicated setup

### Additional Context

https://github.com/bitwarden/server/issues/307

### Build Version

-

### Environment

Self-Hosted

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by notnamed** [comment]
Thank you for your feedback! You can follow any of our installation guides depending on your environment:

Automatic installer - Linux: https://bitwarden.com/help/install-on-premise-linux/
Automatic installer - Windows: https://bitwarden.com/help/install-on-premise-windows/
Manual installation: https://bitwarden.com/help/install-on-premise-manual/
Offline installation - Linux: https://bitwarden.com/help/install-and-deploy-offline/
Offline installation - Windows: https://bitwarden.com/help/install-and-deploy-offline-windows/
Unified (single-container architecture) beta: https://bitwarden.com/help/install-and-deploy-unified-beta/

If you have any questions about any of these methods, please feel free to reach out to support (https://bitwarden.com/contact).
